### ISSUE TITLE
Doesn't handle hex literals

### ISSUE THREAD

**Issue by Graham42** [thorbenvh8/prettier-java#46]
Sample code:
```
public class MyTest {
	public static final int	MY_HEX_THING = 0x0000;
}
```
Output:
```
$ prettier  MyTest.java
Unknown Java node: { type: 'HEX_LITERAL', value: '0x0000' }
public class MyTest {
  public static final int MY_HEX_THING = ;
}
```

**Comment by thorbenvh8** [comment]
fixed in next release

**Comment by thorbenvh8** [comment]
Fixed in 0.0.10!
Thanks for the raising the issue! 👍
